.github/workflows/release-pypi.yml

```diff
@@ -0,0 +1,79 @@
+---
+name: Release to PyPI
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build_sdist:
+    name: Build source package
+    runs-on: 'ubuntu-latest'
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install/Upgrade Python dependencies
+      run: python -m pip install --upgrade pip wheel
+
+    - name: Build and install Python source package
+      run: |
+        python setup.py sdist
+        python -m pip install dist/*.tar.gz
+
+    - name: Run tests
+      run: |
+        pip install pytest pytest-xdist
+        pytest -n auto test/
+
+    - uses: actions/upload-artifact@v2
+      with:
+        path: dist/*.tar.gz
+
+
+  build_bdist:
+    name: Build binary wheels
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # Windows build fails
+        os: [ubuntu-latest, macos-latest]  #  windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    # setup Python for cibuildwheel
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    # for other architectures, see: https://cibuildwheel.readthedocs.io/en/stable/faq/#emulation
+    - name: Build wheels for CPython
+      uses: pypa/cibuildwheel@v2.1.1
+      env:
+        CIBW_SKIP: 'cp310-* pp*'  # 3.10 and pypy builds are broken
+        CIBW_ARCHS: 'auto64'  # only 64-bit
+        CIBW_TEST_REQUIRES: pytest pytest-xdist
+        CIBW_TEST_COMMAND: pytest -n auto {project}/test/
+
+    - uses: actions/upload-artifact@v2
+      with:
+        path: ./wheelhouse/*.whl
+
+
+  upload_pypi:
+    name: Publish packages to PyPI
+    # only publish packages once everything is successful
+    needs: [build_bdist, build_sdist]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/download-artifact@v2
+        with:
+          name: artifact
+          path: dist
+
+      - uses: pypa/gh-action-pypi-publish@v1.4.2
+        with:
+          user: __token__
+          password: ${{ secrets.pypi_api_token }}
+          # To test: repository_url: https://test.pypi.org/legacy/
```

Comment:
I don't understand this comment. Does it mean that currently the builds are uploaded to `test.pypi.org`?

Reply:
No - the default `repository_url` is the correct one, but you can override it with whichever value you like for testing purposes.  

I can remove that comment if it adds no value.

Reply:
aaah ok, *now* I understand what is the point of the comment. I made a suggestion to make it clearer (and this is not a nitpick, I really didn't understand it at first 😅)